scipy/special/special_wrappers.cpp

```diff
@@ -208,9 +216,11 @@ void modified_fresnel_minus_wrap(double x, npy_cdouble *Fminus, npy_cdouble *Kmi
                                     reinterpret_cast<complex<double> *>(Kminus));
 }
 
-double sin_pi(double x) { return special::sin_pi(x); }
+double special_sinpi(double x) { return special::sinpi(x); }
+
+npy_cdouble special_csinpi(npy_cdouble z) { return to_ccomplex(special::sinpi(to_complex(z))); }
 
-double cos_pi(double x) { return special::cos_pi(x); }
+double special_cospi(double x) { return special::cos_pi(x); }
```

Comment:
You seem to be using the `amos` `cospi` here instead of the `cephes` version which is currently the version SciPy uses for its `cospi` ufunc. Unless you've seen empirical evidence otherwise, I think the `cephes` version is superior.

Reply:
I think that's what it is doing before? In any case, switched to the cephes one.

Reply:
https://github.com/scipy/scipy/blob/438e88f5e1752f49b5c760e15c94c48f26d83e4d/scipy/special/functions.json#L12-L19

Reply:
Indeed, so it wasn't.

Though `special::cos_pi` did preexist, https://github.com/scipy/scipy/blob/main/scipy/special/special_wrappers.cpp#L213, no doubt from one of the many different wrappers / whatever.

Anyway, changed.